sections/exchange/TradeCard/Charts/Types/AreaChart.tsx

```diff
@@ -69,27 +82,47 @@ const AreaChart: FC<{
 						<stop offset="100%" stopColor={chartColor} stopOpacity={0} />
 					</linearGradient>
 				</defs>
-				<XAxis
-					// @ts-ignore
-					dx={-1}
-					dy={10}
-					minTickGap={20}
-					dataKey="timestamp"
-					allowDataOverflow={true}
-					tick={fontStyle}
-					axisLine={false}
-					tickLine={false}
-					tickFormatter={(val) => {
-						if (!isNumber(val)) {
-							return '';
-						}
-						const periodOverOneDay =
-							selectedChartPeriodLabel != null &&
-							selectedChartPeriodLabel.value > PERIOD_IN_HOURS.ONE_DAY;
-
-						return formatDate(val, periodOverOneDay ? 'dd MMM' : 'h:mma');
-					}}
-				/>
+				{isPeriodGreaterThanOneDay ? (
```

Comment:
two ternary operators based on same predicate due to Recharts not rendering <Fragment/> elements

Reply:
this is because you had to render day and month axis separately right? 

what happens if there are two months in the x-axis?

Reply:
Yeah, exactly. I'm rendering 1 x-axis as the day, and the second as the month.  Here's a screenshot of 2 months in the same view:
![image](https://user-images.githubusercontent.com/16483341/127250115-7a79d2e2-2a64-49e2-9f3a-3e3e5c34d199.png)

Reply:
Just need to offset that june label to the left